pkgs/applications/misc/plasma-applet-volumewin7mixer/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, cmake, extra-cmake-modules, plasma-framework, kwindowsystem, fetchFromGitHub, }:
+
+let version = "23"; in
+
+stdenv.mkDerivation {
+  name = "plasma-applet-volumewin7mixer";
+
+  src = fetchFromGitHub {
+    owner = "Zren";
+    repo = "plasma-applet-volumewin7mixer";
+    rev = "v${version}";
+    sha256 = "1j2bq343lnhwqz26qfsvg7vjxv84ibzbc4y86rjkh07nqjwb3xsc";
+  };
+
+  patches = [ ./cmake.patch ];
+  postPatch = '' rm build '';
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
+  buildInputs = [ plasma-framework kwindowsystem ];
+
+  meta = with stdenv.lib; {
+    description = "A fork of the default volume plasmoid with a Windows 7 theme (vertical sliders).";
+    homepage = "https://github.com/Zren/plasma-applet-volumewin7mixer";
+    license = licenses.gpl3;
```

Comment:
I can't find a `LICENSE` file in the repository, or some other indication that its licensed under the terms and conditions of the GPL-3.

Reply:
I assumed the license from the packages for other distros mentioned in the README ([archlinux](https://aur.archlinux.org/packages/plasma5-applets-volumewin7mixer/) and [chakralinux](https://chakralinux.org/ccr/packages.php?ID=7763)).

Reply:
After a search on what GPL means on archlinux, it should probably be gpl2Plus.

Reply:
Since this package is a fork of the original volume mixer applet of the plasma desktop (which is GPL2), I guess the license should stay the same right? I'll open an issue in the original project to confirm what the license is, to be sure.

Reply:
https://github.com/Zren/plasma-applet-volumewin7mixer/issues/3#issuecomment-418795091 Yep, this is GPL2.